src/main/resources/db_scripts/db_script_v0.sql

```diff
@@ -0,0 +1,45 @@
+create table "workflow" (
+  "name" VARCHAR(45) NOT NULL UNIQUE,
+  "is_active" BOOLEAN NOT NULL,
+  "created" TIMESTAMP NOT NULL,
+  "updated" TIMESTAMP,
+  "id" BIGSERIAL NOT NULL PRIMARY KEY
+);
+
+create table "job_instance" (
+  "job_name" VARCHAR NOT NULL,
+  "job_definition_id" BIGINT NOT NULL,
+  "event_id" VARCHAR(70) NOT NULL UNIQUE,
+  "job_type" VARCHAR NOT NULL,
+  "variables" VARCHAR NOT NULL,
+  "maps" VARCHAR NOT NULL,
+  "job_status" VARCHAR NOT NULL,
+  "executor_job_id" VARCHAR,
```

Comment:
How is it that the executor might by null?

Reply:
It is there for case that for exampe Yarn will not give you id before the job is accepted. So there is a few "seconds" without id. (and in future some executors will have no id, for example email sending job.)